superset-frontend/plugins/plugin-chart-table/src/buildQuery.ts

```diff
@@ -198,10 +198,15 @@ const buildQuery: BuildQuery<TableChartFormData> = (
         (ownState.currentPage ?? 0) * (ownState.pageSize ?? 0);
     }
 
+    if (!temporalColum) {
+      // This query is using only textual columns, so it doesn't need time grain
```

Comment:
Or numerical presumably. Probably most accurate just to say it doesn't have any temporal columns so there's no time grain.